packages/admin/src/components/shared/SelectInfinite/SelectInfinite.tsx

```diff
@@ -58,13 +60,9 @@ const SelectInfinite = (props: SelectInfiniteProps): JSX.Element => {
       return field
     }
 
-    if (typeof field.value !== 'object' || field.value.id === undefined || field.value.name === undefined) {
-      throw new Error('The source data must contain the "id" and "name" properties')
-    }
-
     return {
       ...field,
-      value: data ? field.value.id : ''
+      value: data ? field.value?.id ?? '' : ''
```

Comment:
```suggestion
      value: data && field.value ? field.value?.id : ''
```
Если id всегда будет.

Reply:
Там id не всегда будет. Когда это поле используется при создании записи, я начальный id не знаю, поэтому мне здесь нужно пустую строку отправить в значение поля. Немного по дурацки получилось, но как есть)